pkg/exporter/sumologicexporter/config.go

```diff
@@ -135,4 +140,6 @@ const (
 	DefaultClient string = "otelcol"
 	// DefaultGraphiteTemplate defines default template for Graphite
 	DefaultGraphiteTemplate string = "%{_metric_}"
+	// DefaultTransformMetadata defines default TransformMetadata
```

Comment:
```suggestion
	// By default TransformMetadata is enabled
```

Reply:
The comment I added matches all the other comments in this struct if you look at the other fields like `Default Compress`, `DefaultLogFormat` etc.